guides/common/modules/proc_deploying-a-virt-who-configuration-on-a-separate-rhel-server.adoc

```diff
@@ -0,0 +1,53 @@
+[id="deploying-a-virt-who-configuration-on-a-separate-rhel-server"]
+= Deploying a virt-who configuration on a dedicated {EL} server
+
+For VMware vSphere, Microsoft Hyper-V, and Nutanix AHV, you can deploy the virt-who configuration on a dedicated {EL} server.
+The server can be physical or virtual.
+
+Global configuration values apply to all virt-who configurations on this server and are overwritten if you deploy a new virt-who configuration.
+
+.Prerequisites
+* You have registered the {EL} server to {ProjectServer}. For more information, see {ProvisioningDocURL}registering_hosts_to_server_managing-hosts#Registering_Hosts_by_Using_Global_Registration_managing-hosts[Registering hosts by using global registration] in _{ProvisioningDocTitle}_.
+
+.Procedure
+. Add a TCP port for communication between virt-who and {ProjectServer} to the runtime configuration:
++
+[options="nowrap"]
+----
+$ firewall-cmd --add-port="<port>/tcp" <1>
+----
+<1> Specify `443` for VMware vCenter and Nutanix AHV. Specify `5985` for Microsoft Hyper-V.
+. Add a TCP port for communication between virt-who and {ProjectServer} to the permanent configuration:
++
+[options="nowrap"]
+----
+$ firewall-cmd --add-port="<port>/tcp" --permanent
+----
+. In the {ProjectWebUI}, navigate to *Infrastructure* > *Virt-who Configurations*.
```

Comment:
Again, this is very similar to the other procedures. It makes me wonder if we could enhance virt-who to utilize REX.